membership-common/src/test/scala/com/gu/memsub/subsv2/services/CatalogServiceTest.scala

```diff
@@ -33,7 +34,7 @@ class CatalogServiceTest extends Specification {
 object CatalogServiceTest {
 
   def client(path: String) = {
-    val runner = (r: Request) =>
+    val runner: HttpClient[Id] = (r: Request) =>
```

Comment:
giving it a type here explicitly makes scala convert the function to the trait automatically (single abstract method)